core/src/main/java/io/stargate/core/metrics/api/Metrics.java

```diff
@@ -36,4 +37,12 @@
    * <p>Note that this registry will warp all the metrics reported via {@link #getRegistry()}.
    */
   MeterRegistry getMeterRegistry();
+
+  /**
+   * Returns set of default micrometer {@link Tags} for the given module.
+   *
+   * @param module module name
+   * @return Tags
+   */
+  Tags defaultTagsForModule(String module);
```

Comment:
nit: rename to `tagsForModule`? Just shorter and it's obvious that one can add more tags later. WDYT?

Reply:
done, resolving..